src/components/custom/_dhero-dmt5.js

```diff
@@ -175,11 +177,21 @@ const DHero = ({
                     </StyledHeaderTitle>
                 </HeroContent>
                 <LinkWrapper>
-                    {join_us_for_free && (
-                        <DemoButton onClick={handleSignup} id="dm-hero-signup-1" secondary>
-                            {localize('Create free demo account')}
-                        </DemoButton>
-                    )}
+                    {join_us_for_free &&
+                        (is_logged_in ? (
+                            <DemoButton
+                                onClick={handleGetTrading}
+                                id="dm-hero-signup"
```

Comment:
The id should be this, when it's not a signup button
```suggestion
                                id="dm-hero-signup"
```